bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -2680,6 +2680,24 @@
             insert into order_type_class_map(order_type_id, concept_class_id) values(@order_type_id,@lab_test);
         </sql>
     </changeSet>
+    <changeSet id="bahmni-core-20221206644-bah-2652" author="clinic-config">
```

Comment:
Prefer splitting them in separate changeset. And please use preconditions. we shouldn't assume that the required entities exist or any mapping already does not exist.

Reply:
Updated the change set Angshu please review.